### PR TITLE
add redis sentinel configuration v2.2

### LINKED ISSUE #1140: Add support for a high availability redis
At the moment openSRP only supports redis standalone setup which does not offer high availability or scaling features. To increase the availability of redis we need to add support for redis sentinel or redis cluster since they both support automatic failover incase one of its nodes goes down. However on top of automatic failover redis cluster offers scaling with data sharding which would be suitable for handling larger data and providing faster access to it.

### PR DESCRIPTION
**Pull request by bennsimon**
- closes #1140 
- Add support for no password redis configuration.

### CODE REVIEW

**Comment by bennsimon** [comment]
Linked PR https://github.com/opensrp/opensrp-server-configs/pull/151

**Comment by ndegwamartin** [comment]
@bennsimon looks good, needs an update with master branch. I think we can also consider adding some type of documentation on the Read ME for this.

File: README.md [hunk 1]
```diff
@@ -154,3 +154,20 @@ jvm_threads_states_threads{state="blocked",} 0.0
...
+### Redis for High Availability
+
+Redis Sentinel support is added on openSRP server web. To enable redis sentinel the following configurations were added:
+
+#### Configurations for the redis sentinel
+
+| Configuration        | Description                                                                     | Type                         | Default      |
+|----------------------|---------------------------------------------------------------------------------|------------------------------|--------------|
+| redis.sentinels      | Comma separated string of redis sentinel e.g. "localhost:26379,localhost:26380" | String                       | ""           |
+| redis.master         | Name of the set of redis instances to monitor.                                  | String                       | "mymaster"   |
```
Comment (ndegwamartin) [inline README.md:167]: Could we add another column for _other options available_? or add them in the text at the bottom e.g. I see we have my master as the default or `redis.master` but what are alternative options? 

Also is that a name on AWS or IP address type entry e.t.c

Comment (bennsimon) [reply README.md]: @ndegwamartin Updated.

**Review by ndegwamartin — APPROVED** [review]

**Pull request merged**